benchmarks/TerraFX.Interop.Mimalloc.AllocatorsBenchmarks.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Missing `<?xml version="1.0" encoding="utf-8"?>` header that makes this correctly detected by default in various editors.